install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -349,6 +350,12 @@ data:
   #  - portmap (Enables HostPort support for Cilium)
   cni-chaining-mode: {{ .Values.global.cni.chainingMode }}
 
+{{- if hasKey .Values "enableIdentityMap"}}
+  enable-identity-map: {{ .Values.global.enableIdentityMap | quote }}
```

Comment:
Don't we have a mismatch here? The new argument in the daemon is `enable-identity-mark`, but here we define `enable-identity-mark`. Shouldn't we do `s/map/mark/` in helm charts?

Am I missing something here?

I noticed that when backporting to 1.7.

Reply:
Ah, sorry, it got fixed here https://github.com/cilium/cilium/pull/12194